### DISCUSSION TITLE
Jquery - swipebox - problems with displaying the buttons  .close  . prev.  .next

### DISCUSSION THREAD

**Discussion by Noturno411** [community/community#106838] [Programming Help] [answered]
### Body

Sorry for any wrong post types, tags etc.

Hello, I'm experiencing a problem that I can't resolve. I made a gallery in swipebox. I made an image file called icons.png with the button designs ( X < > ) and called it in my code. The buttons are repeating themselves (it's becoming X X X ) or they should be the arrows on the right side. Can anyone help me because these X's are repeating and none of the icons.png files are drawn correctly? Thank you very much.

Here's an excerpt from my swipebox.css file (if you want the whole thing, I'll send it) and a screenshot of the screen it's printing.

#swipebox-action #swipebox-previous,
#swipebox-action #swipebox-next,
#swipebox-action #swipebox-close {
background image: url(../images/icons.png);
background repeat: no repeat;
border: none !important;
text decoration: none! Important;
cursor: pointer;
position: absolute;
width: 50px;
height: 50px;
top: 0;
}

#swipebox-action #swipebox-close {
background-position: 15px 12px 0 auto;
left: 40px;
}

#swipebox-action #swipebox-prev {
background-position: 15px 25px 0 auto;
right: 100px;
}

#swipebox-action #swipebox-next {
background-position: -38px 13px 0 auto;
right: 40px;
}

#swipebox-action #swipebox-prev.disabled,
#swipebox-action #swipebox-next.disabled {
filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=30);
opacity: 0.3;
}

[SCREEN] - note that there are some dots too, it looks buggy. Strange because calling an image should show exactly the image. Could it be some configuration in my swipebox.css that would be repeating these X X X?
<img width="373" alt="screen" src="https://github.com/community/community/assets/159219120/0a6cc81e-30b5-47a1-b546-426645bc79cb">

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by gil-son** [comment]
It looks like there might be a syntax error in your CSS. The background property should have url and repeat specified in the same line, and there's a missing colon after background in your code. Also, it seems like there's a typo in #swipebox-prev, where it's written as #swipebox-previous. Example:

```
#swipebox-action #swipebox-previous,
#swipebox-action #swipebox-next,
#swipebox-action #swipebox-close {
    background: url(../images/icons.png) no-repeat;
    border: none !important;
    text-decoration: none !important;
    cursor: pointer;
    position: absolute;
    width: 50px;
    height: 50px;
    top: 0;
}

#swipebox-action #swipebox-close {
    background-position: 15px 12px 0 auto;
    left: 40px;
}

#swipebox-action #swipebox-previous {
    background-position: 15px 25px 0 auto;
    right: 100px;
}

#swipebox-action #swipebox-next {
    background-position: -38px 13px 0 auto;
    right: 40px;
}

#swipebox-action #swipebox-previous.disabled,
#swipebox-action #swipebox-next.disabled {
    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=30);
    opacity: 0.3;
}

```

and

check the path of the image is correct

**Reply by Noturno411** [reply]
Thanks for helping.
I did what you said and now it's like this.

There was an X on each side.

I checked the image path, it is also correct.

Please help me. That would be very strange.

I was going to comment that it could be the image, but it doesn't make sense. The screen shows two x's and the image will be X < >

Look below how it looks now
<img width="373" alt="screen" src="https://github.com/community/community/assets/159219120/fb2b25b7-79f4-450b-a07c-2808e155b0c9">

**Reply by gil-son** [reply]
Try to use other icon just for test

**Reply by Noturno411** [reply]
I will test, 

I adjusted the background-position of the previous and next in the swipebox.css file and several arrows appeared and it only worked to the right. Something very wrong is happening

**Reply by gil-son** [reply]
You need to organize the responsive of the screen

This [documentation](https://www.w3schools.com/css/css_rwd_intro.asp) show how do it

If I supported you consider to mark my answer

**Reply by Noturno411** [reply]
thanks buddy!

I'm afraid of changing the responsive file and disconfiguring the entire site.

You helped a lot, but I wanted to see if anyone could come up with any new ideas to solve this.

I was checking the default swipebox css and js file and they are defining the button name as .prev.

Can you tell me If I mark your answer as a solution, will the moderator close the topic?

**Reply by gil-son** [reply]
Can you share the link of your repository?

I will check it

**Reply by Noturno411** [reply]
Sure. Thanks for the support. See if you can open the link.

https://github.com/Noturno411/unique.git

**Reply by gil-son** [reply]
I opened duq-central-houses.html on 2 different browsers (Chrome and Firefox) and I didn't find the problem:

Firefox:

![support01](https://github.com/community/community/assets/72712095/1eef38ee-a99f-4bd6-bd08-775c05377d76)


Google Chrome:

![support02](https://github.com/community/community/assets/72712095/f58f10bb-4022-46c5-9b8e-8a768751785d)

**Reply by Noturno411** [reply]
not be that image.

Scroll down and open images from photo gallery.

where it says "plant" and "images of the project".

the problem with the buttons is with these images.

**Reply by gil-son** [reply, marked as answer]
I understood now

You have to create a different scope/object to each icon and use a different image, like that:

#swipebox-next
#swipebox-prev
#swipebox-close


I will give you an optional solution. Replace that script:
```
#swipebox-next {
  background-position: -38px 15px;
  float: right;
  right: 50px;
}
```
to that other script:
```
#swipebox-next {
  background-position: -38px 15px;
  float: right;
  right: 50px;
  -webkit-transform: rotate(180deg);
  -moz-transform: rotate(180deg);
  -o-transform: rotate(180deg);
  -ms-transform: rotate(180deg);
  transform: rotate(180deg);
}
```
It will rotate the previous image on 180 degrees

You can speak Portuguese language?

So, if I supported you, consider to mark my answer

**Reply by Noturno411** [reply]
yes I'm brazilian.

I changed it to the script you mentioned and it didn't work. no effect. I'll mark it as an answer. Thanks!

**Reply by gil-son** [reply]
call me on Discord: gil_son

I am brazilian too
